infrastructure/ansible/roles/ats/tasks/ats.yml

```diff
@@ -53,12 +53,22 @@
     state: stopped
   ignore_errors: true
 
+- name: Create tmp dir for t3c log
+  file:
+    path: /tmp/trafficcontrol-cache-config
+    state: directory
+    recurse: yes
+    owner: root
+    group: root
+    mode: 0755
+
 - name: Run ORT Badass
-  shell: >-
+  shell:
```

Comment:
You're missing the executable flag to force it to use bash.  This is needed because `PIPESTATUS` is a bash-ism.

Reply:
Fixed

Reply:
This isn't at the place.  It's part of the shell module, not the task.

Reply:
Ah, moved